dotnet/dotnet.lua

```diff
@@ -0,0 +1,171 @@
+---    Copyright 2024 [liquidiert haberederkorbinian@googlemail.com]
+--
+--   Licensed under the Apache License, Version 2.0 (the "License");
+--   you may not use this file except in compliance with the License.
+--   You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+--   Unless required by applicable law or agreed to in writing, software
+--   distributed under the License is distributed on an "AS IS" BASIS,
+--   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--   See the License for the specific language governing permissions and
+--   limitations under the License.
+
+local http = require("http")
+local json = require("json")
+local html = require("html")
+
+OS_TYPE = ""
+ARCH_TYPE = ""
+
+BARE_URL = "https://dotnet.microsoft.com"
+VERSIONS_URL = "https://dotnet.microsoft.com/en-us/download/dotnet"
+
+--- https://go.dev/dl/go1.21.5.darwin-arm64.tar.gz
```

Comment:
Need to delete this line, should be missed. ;)

Reply:
Oh yeah forgot it after copying :D